locales/en-US/server.ftl

```diff
@@ -185,6 +185,8 @@ annotationUndoButton =
     .title = Undo
 annotationRedoButton =
     .title = Redo
+annotationTextButton =
+    .title = Text
```

Comment:
Is there a mockup of this feature? I'm having a hard time picturing this tooltip.

Could "Annotate", "Add text" be a better choice?

Reply:
I prefer "Add text" here

Reply:
@flodolo here are links to screenshot of TextTool button in editor and TextTool edit screen seen on click of text button
https://imgur.com/a/SyMIKBv
https://imgur.com/a/n3kWAaO